examples/uefi-demo/main.rs

```diff
@@ -11,14 +11,32 @@ use alloc::{
     format,
     rc::Rc,
     string::{String, ToString},
+    vec,
+};
+use core::{
+    slice,
+    sync::atomic::{AtomicPtr, Ordering},
+    time::Duration,
+};
+
+use log::info;
+use uefi::{
+    prelude::*,
+    proto::console::{gop::BltPixel, pointer::Pointer},
+    table::boot::ScopedProtocol,
+    Char16,
 };
-use core::{slice, time::Duration};
-use slint::{platform::software_renderer, SharedString};
-use uefi::{prelude::*, proto::console::gop::BltPixel, Char16};
 use uefi_services::system_table;
 
+use slint::{platform::software_renderer, SharedString};
+
 slint::include_modules!();
 
+static POINTER: AtomicPtr<ScopedProtocol<'static, Pointer>> = AtomicPtr::new(core::ptr::null_mut());
```

Comment:
this could probably be named MOUSE_POINTER because i thought this was because it was a memory pointer (AtomicPtr) but that was misleading.